src/applications/disability-calculator/components/RatingRow.jsx

```diff
@@ -50,18 +49,17 @@ const RatingRow = ({
         />
       </div>
       <div className="vads-l-col--3">
-        {canDelete && (
-          <div>
-            <button
-              type="button"
-              onClick={handleRemoveRating(indx)}
-              className="va-button-link delete-btn vads-u-margin--1p5"
-            >
-              <i className="fas fa-trash-alt vads-u-padding-right--0p5" />
-              Delete
-            </button>
-          </div>
-        )}
+        <div>
+          <button
+            type="button"
+            onClick={handleRemoveRating(indx)}
+            className="va-button-link delete-btn vads-u-margin--1p5"
+            disabled={canDelete < 3}
```

Comment:
Following up on the previous comment in `DisabilityRatingCalculator.jsx`, change the prop from `canDelete` to `disabled` and set `disabled={disabled}`